manual/mailings/call_for_papers.rst

```diff
@@ -15,19 +16,22 @@ scientific computing in Python, such as:
 - hardware systems designed to support the Python stack
 - applications of scientific computing in education
 
-Papers should be submitted as pull requests by May 30th.  At that point, the 
-two-week open review period will begin.  During this time, members from the 
-program committee and the scientific Python community at large may provide 
+Papers must be submitted as pull requests by May 14th.  At that point, the 
+six-week open review period will begin.  During this time, members from the 
+program committee and the scientific Python community at large will provide 
 feedback, which should be addressed through discussion on GitHub and by 
 updating the pull request.
 
+When submitting a paper to the proceedings, you may be contacted by the
+Proceedings Chairs to assist in reviewing one paper.
```

Comment:
I think we should ask for …say up-to three papers? that way we can guarantee that each paper will have at least 3 reviews (not including the editor's metareview)?